docs/why_mf_next.md

```diff
@@ -68,34 +79,45 @@ But the stability requirements prevent any major cleanup.
 ### 2. Some existing problems
 
 - ICU added new formatters, but MessageFormat does not support them
-- Combined selectors (select + plural) results in unreadable and error
-  prone nesting
-- Select and plurals inside the message are difficult to translate because of
-  grammatical agreement requires words outside select / plural to change.
+- Messages with selectors (`select` and/or `plural`) are difficult to create
+  and edit because of the complex nesting requirements of the syntax.
+- `select` and `plural` placeholders inside a message are difficult to translate as
```

Comment:
```suggestion
- `select` and `plural` placeholders inside a message are difficult to translate as
  grammatical agreement may require words _outside_ the `select`/`plural` to change.
```
Remove this item since it is covered above. (I wouldn't weigh too heavily on this, as systems can enforce "no select on parts", as Google does.)

Reply:
They aren't the same thing. Nesting happens when you don't do select on parts but do have multiple selection. Coupled with the need to "explode" plurals, things can get messy. (At Amazon we used our resource format to break up the nesting, making it easier to write).

This second bullet is about naive use of MF1 (avoiding nesting works in English, but doesn't translate). You or I can enforce this in our own tooling, but MF1 itself does not.

In any case, I think this is fine?

Reply:
Ok for now, thanks.